docs/website/developers/committers/release-process/verify-release-artifacts.md

```diff
@@ -36,10 +36,12 @@ curl https://dist.apache.org/repos/dist/release/incubator/brooklyn/KEYS | gpg2 -
 {% endhighlight %}
 
 
-Verify the hashes and signatures of artifacts
----------------------------------------------
+Download the artifacts
+----------------------
 
-If the releases have been published to the pre-release area, download them:
+If you've just built the RC, simply go to that directory and skip this step.
+
+If you're verifying a build someone else has made, first download the files including all keys using:
```

Comment:
I'd recommend that even the RM downloads the files, to confirm that the published URLs are correct and working.